src-dict/src-clean.txt

```diff
@@ -71371,6 +71415,7 @@ electrolier=noun=all
 electrologist=noun=all
 electroluminescence=noun=all
 electroluminescent=JJ=all
+electrolyze|electrolyse=verb=us|au,ca,gb,nz,za
```

Comment:
We are using these variants for -ize/-ise:
> prioritize|prioritise=verb=us,ca,gb|gb,au,nz,za
> psychoanalyze|psychoanalyse=verb=ca,us,gb|au,gb,nz,za

The -z- in GB is the Oxford spelling that can be enabled/disabled with a user option.

Reply:
Thanks. Just fixed that, as well as a typo that I made.